__tests__/app.test.js

```diff
@@ -71,8 +71,70 @@ describe("GET /api/reviews", () => {
         });
       });
   });
-});
+  test("only returns reviews of a given category when queried by category", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review.category).toBe("dexterity");
+        });
+      });
+  });
+  it("should return an empty array if the given category query has no reviews", () => {
+    return request(app)
+      .get("/api/reviews?category=test")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toEqual([]);
+      });
+  });
+  test("sorts by a given column name if the sort_by query is provided", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=comment_count")
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted({ key: "comment_count", descending: true });
+      });
+  });
+  test("orders results correctly when the order query is provided", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=votes&order=ASC")
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted({ key: "votes", descending: false });
+      });
+  });
 
+  it("should ignore invalid values for sort_by", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=test")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted({ key: "created_at", descending: true });
+      });
+  });
+  it("should ignore invalid values for order", () => {
+    return request(app)
+      .get("/api/reviews?order=test")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSorted({ key: "created_at", descending: true });
+      });
+  });
+  it("should ignore extra queries if provided", () => {
```

Comment:
I like what you have done here but this test would have likely passed first time, I dont know if its needed as such